packages/cadl-ranch-specs/http/special-headers/repeatability/main.tsp

```diff
@@ -0,0 +1,39 @@
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/cadl-ranch-expect";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+@doc("Illustrates OASIS repeatability headers")
+@supportedBy("dpg")
+@scenarioService("/special-headers/repeatability")
+namespace SpecialHeaders.Repeatability;
+
+model RepeatableResponse {
+  @doc("The status code.")
+  @statusCode
+  statusCode: 204;
+
+  @visibility("read")
+  @header("Repeatability-Result")
+  @doc("Indicates whether the repeatable request was accepted or rejected.")
+  repeatabilityResult?: "accepted" | "rejected";
```

Comment:
reason we need this? Wasn't the NoContent all we needed?

Also this feels invalid returning 204 with a body